sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/TriggerExecutor.scala

```diff
@@ -65,8 +65,22 @@ case class ProcessingTimeExecutor(processingTime: ProcessingTime, clock: Clock =
       s"${intervalMs} milliseconds, but spent ${realElapsedTimeMs} milliseconds")
   }
 
-  /** Return the next multiple of intervalMs */
+  /** Return the next multiple of intervalMs
```

Comment:
Nit: comment style is off, we use [javadoc style](https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Codedocumentationstyle)